baselines/pyproject.toml

```diff
@@ -37,13 +37,17 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8.3"
+python = "^3.7.12"
 # Mandatory dependencies
-flwr = "^0.17.0"
-# flwr = { path = "../", develop = true }  # For development
+#flwr = "^0.17.0"
+flwr = { path = "../", develop = true }  # For development
 torch = "^1.10.1"
 torchvision = "^0.11.2"
+omegaconf = { git = "https://github.com/pedropgusmao/omegaconf.git" } 
+hydra-core = { git = "https://github.com/pedropgusmao/hydra.git", branch="main" }
```

Comment:
`hydra-core` and `omegaconf` deps make the process of `poetry install` hang in an endless _resolving dependencies_ loop. If i comment these two lines out, `poetry install` runs (but ofcourse these two deps are not installed)

Reply:
Nevermind. It eventually ended... I found it faster to do first `poetry install` without those two deps. Then add them back and do `poetry update`....

Reply:
In my machine, it takes 1205.3s, which I think is way too long. I will have to dig deeper on this issue, but I think it could be a different PR.